libbeat/processors/add_host_metadata/add_host_metadata.go

```diff
@@ -96,7 +97,7 @@ func New(cfg *config.C) (beat.Processor, error) {
 	}
 
 	// create a unique ID for this instance of the processor
-	cbIDStr := ""
+	var cbIDStr string
```

Comment:
This change is unrelated to this PR; it is included in this PR to make the linter happy.  Without it, the linter was failing with this error:

```
assigned to cbIDStr, but reassigned without using the value (wastedassign)
```